source/index.html

```diff
@@ -5,8 +5,198 @@
+        <div class="get-app__download app-download">
+          <a href="https://www.apple.com/app-store/" class="app-download__link" target="_blank">Скачать приложение</a>
+          <div class="app-download__store app-store">
+            <ul class="app-store__list">
+              <li class="app-store__item">
+                <a href="https://www.apple.com/app-store/" class="app-store__link app-store__link--app-store" target="_blank">
+                  <span class="visually-hidden">Скачать в App Store</span>
+                </a>
+              </li>
+              <li class="app-store__item">
+                <a href="https://play.google.com/store/" class="app-store__link app-store__link--play-market" target="_blank">
+                  <span class="visually-hidden">Скачать в Play Market</span>
+                </a>
+              </li>
+              <li class="app-store__item">
+                <a href="https://www.microsoft.com/" class="app-store__link app-store__link--ms-store" target="_blank">
+                  <span class="visually-hidden">Скачать в MicroSoft Store</span>
+                </a>
+              </li>
+            </ul>
+            <p class="app-store__desc">Доступно для iPhone, iPad, Android, Windows Phone, OS X, Windows 8</p>
+          </div>
+        </div>
+      </section>
+      <section class="how-work">
+        <h2 class="visually-hidden">Что делает приложение</h2>
+        <div class="how-work__container container">
+          <p class="how-work__title title">Устали от серости мегаполиса?<br>Нам есть, что вам предложить!</p>
+          <ul class="how-work__list">
+            <li class="how-work__item">
+              <h3 class="how-work__subtitle">Поднимает настроение</h3>
+              <p class="how-work__text">Приложение позволит вам победить осеннюю хандру и депрессию буквально в несколько кликов!</p>
+            </li>
+            <li class="how-work__item">
+              <h3 class="how-work__subtitle">Меняет мир вокруг</h3>
+              <p class="how-work__text">Сделайте снимок и украсьте его смайликом или текстовой подписью, чтобы усилить эффект</p>
+            </li>
+            <li class="how-work__item">
+              <h3 class="how-work__subtitle">Заводит новых друзей</h3>
+              <p class="how-work__text">Ставьте лайки, комментируйте, делитесь фотографиями с друзьями и заводите новых</p>
+              <img src="https://dummyimage.com/136x245.gif" width="136" height="245" alt="В телефоне запущено приложение Pink" class="how-work__image">
+            </li>
+          </ul>
+        </div>
+      </section>
+      <div class="time">
+        <div class="time__container container">
+          <p class="time__title">Эффект на 24 часа!</p>
+          <p class="time__desc">Британские ученые провели исследования и&nbsp;доказали, что положительный эффект от&nbsp;использования приложения длится целые сутки!</p>
+          <a href="#" class="time__link">Узнать больше об исследовании</a>
```

Comment:
давай единообразия ради указывать `class` в начале, иначе при чтении сбивается вертикальный ритм кода
```suggestion
          <a class="time__link" href="#">Узнать больше об исследовании</a>
```